meta/labs/README.md

```diff
@@ -1,53 +1,19 @@
 # Labs
 
-This document will contain a brief overview of our labs program and what it'll
-entail going into the 2015 fall semester.
+We're working with 12 clubs to build the first version of our club playbook (in
+this repository!) in the fall 2015 semester. Labs is what we're calling the 12
+clubs we've selected.
 
-## Format
+Labs clubs are led by the vehement Lab Directors. They're the brave souls that
```

Comment:
`vehement`

Interesting word choice. There are other words I  would use here, but I can't find them tonight. Let's go with it